.github/workflows/bench_upload_for_pr.yaml

```diff
@@ -0,0 +1,86 @@
+name: Benchmark upload
+
+#
+# This workflow runs after every benchmark, to uploading the report to S3.
+#
+# It is it's own workflow, because PRs can contain malicious code and run in a context with secrets..
```

Comment:
```suggestion
# This workflow runs after every benchmark, to upload the report to S3.
#
# It is its own workflow, because PRs can contain malicious code and run in a context with secrets..
```